src/game/animation/updateBall.ts

```diff
@@ -221,3 +222,25 @@ export function updateBallDirectionByCollisionWithBlocks(
   }
   return { x: currentBallDirection.x, y: currentBallDirection.y }
 }
+
+export function updateBallPositionBy(ball: Ball, velocity: Vector): void {
+  const currentBallPosition = getBallCenterPosition(ball)
+  Object.assign(ball.style, {
+    transform:
+    `translate(` +
+      `${currentBallPosition.x - ballSetting.radius + velocity.x}px, ` +
+      `${-(currentBallPosition.y - ballSetting.radius + velocity.y)}px` +
+      `)`
+  })
+}
+
+// TODO: Also available for Bar?
+export function updateBallPositionTo(ball: Ball, position: { x: number, y: number }): void {
+  Object.assign(ball.style, {
+    transform:
+    `translate(` +
+      `${position.x}px, ` +
+      `${position.y}px` +
+      `)`
+  })
+}
```

Comment:
Thanks, it's nicely comprehesive!!! :)

Reply:
Fixed with https://github.com/canalun/brick-break-anywhere/pull/15/commits/f29abee556dcc81ffeaaf2e11d6206ee7ea9d3c4.